test/docfx.Test/DocfxTest.cs

```diff
@@ -206,30 +235,67 @@ private static async Task RunLanguageServer(string docsetPath, DocfxTestSpec spe
 
         private static void RunBuild(string docsetPath, string outputPath, bool dryRun, bool singleFile, DocfxTestSpec spec, Package package)
         {
-            var randomOutputPath = Path.ChangeExtension(outputPath, $".{Guid.NewGuid()}");
+            var randomNormalOutputPath = Path.ChangeExtension(outputPath, $".normal.{Guid.NewGuid()}");
```

Comment:
If `randomContinueOutputPath` is set to `randomOutputPath`, we probably only need one additional `randomIntermediatePath` for 2 stage build.